### DISCUSSION TITLE
Uncompleted Lap Telemetry

### DISCUSSION THREAD

**Discussion by w-i-z-a-r-d** [theOehrly/Fast-F1#124] [Q&A] [answered]
hello, how to get the telemetry data from a specific car when the car retires on that lap, meaning the lap is not completed and the car stops on the track.
 Is there a way to get the partial telemetry?

**Comment by theOehrly** [comment, marked as answer]
The manual way is described here, for example: https://github.com/theOehrly/Fast-F1/issues/41#issuecomment-917413066
